R/combo.R

```diff
@@ -207,3 +207,43 @@ convert_combo_field_to_assay <- function(field) {
   checkmate::assert_string(field)
   DATA_COMBO_INFO_TBL[name == field, ][["type"]]
 }
+
+
+#' Define matrix grid positions
+#'
+#' @param conc1 drug_1 concentration
+#' @param conc2 drug_2 concentration
+#'
+#' @details
+#' \code{drug_1} is diluted along the rows as the y-axis and
+#' \code{drug_2} is diluted along the columns and will be the x-axis.
+#' 
+#' @keywords combination_data
+#' @return list with axis grid positions
+#' 
+#' @export
+define_matrix_grid_positions <- function(conc1, conc2) {
+  .generate_gap_for_single_agent <- function(x) {
+    2 * x[2] - x[3] - log10(1.5)
+  } 
+  
+  conc_1 <- sort(unique(gDRcore::round_concentration(conc1)))
```

Comment:
Let's also move this function to gDRutils (to avoid circular dependency).

Reply:
Done.